File: Carnaval/src/app/Models/totalizacao.ts
export class Totalizacao {

  private _id: number;
  private _detalhe: string;
  private _quantidade: number;
  private _data: Date;
  private _cor: string

  public get Id(): number {
    return this._id;
  }
  public set Id(value: number) {
    this._id = value;
  }
  public get Cor(): string {
    return this._cor;
  }
  public set Cor(value: string) {
    this._cor = value;
  }
  public get Detalhe(): string {
    return this._detalhe;
  }
  public set Detalhe(value: string) {
    this._detalhe = value;
  }
  public get Data(): Date {
    return new Date(this._data);
  }
  public set Data(value: Date) {
    this._data = value;
  }
  public get Quantidade(): number {
    return this._quantidade;
  }
  public set Quantidade(value: number) {
    this._quantidade = value;
  }


}

File: SOM.API/Models/Totalizacao.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SOM.API.Models
{
    [Serializable]
    public class Totalizacao
    {
        private long _id;
        private string _detalhe;
        private string _cor;
        private DateTime _data;
        private int _quantidade;

        public long Id { get => _id; set => _id = value; }
        public string Detalhe { get => _detalhe; set => _detalhe = value; }
        public string Cor { get => _cor; set => _cor = value; }
        public DateTime Data { get => _data; set => _data = value; }
        public int Quantidade { get => _quantidade; set => _quantidade = value; }
    }
}